### PR TITLE
I18n and extras

### PR DESCRIPTION
**Pull request by YesmanYasir**
MEOW MEOWWW MEOW MEOW MEOW

### CODE REVIEW

File: front-end/components/language/Language.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+        <>
+            <Select
+                id="language"
+                options={options}
+                value={selectedOption || null} // Ensure a fallback value of null
+                onChange={handleLanguageChange}
+                className="text-black"
+                classNamePrefix="react-select"
+                isSearchable={false} // Set to true if you want to allow searching
```
Comment (RoelCrabbe) [inline front-end/components/language/Language.tsx]: Remove comments

File: front-end/components/users/UserSignupLoginForm.tsx [hunk 2]
```diff
@@ -122,65 +119,74 @@ const UserSignupLoginForm: React.FC<Props> = ({ isSignUp, onSubmit, clearParentE
 
     return (
         <div className="flex justify-center items-center">
+            <h3 className="px-0">{isSignUp ? t('signup.title') : t('login.title')}</h3>
```
Comment (RoelCrabbe) [inline front-end/components/users/UserSignupLoginForm.tsx]: Why?

File: front-end/package.json [hunk 3]
```diff
@@ -13,20 +13,20 @@
         "bootstrap": "5.2.3",
```
Comment (RoelCrabbe) [inline front-end/package.json]: Pak altijd de latest versions

File: front-end/pages/_app.tsx [hunk 4]
```diff
@@ -1,10 +1,13 @@
 import '@styles/globals.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { appWithTranslation } from 'next-i18next';
 import type { AppProps } from 'next/app';
 
-export default function App({ Component, pageProps }: AppProps) {
```
Comment (RoelCrabbe) [inline front-end/pages/_app.tsx:4]: const app?

File: front-end/pages/index.tsx [hunk 5]
```diff
@@ -17,3 +20,11 @@ const Home: React.FC = () => {
 };
 
 export default Home;
+export const getServerSideProps = async (context: { locale: any }) => {
```
Comment (RoelCrabbe) [inline front-end/pages/index.tsx]: normaal zegt ge beter uwe serverside props boven export default

**Comment by YesmanYasir** [comment]
last comit before merge pr is ready for you to check lmk if its ok is

**Comment by YesmanYasir** [comment]
nu slapen morge vroeg nog beginnen .

**Pull request merged**